lib/vet360/reference_data/response.rb

```diff
@@ -4,11 +4,37 @@
 
 module Vet360
   module ReferenceData
-    class Response < Vet360::Response
-      attribute :reference_data, Array[Hash]
+    class CountriesResponse < Response
+      attribute :countries, Array[Hash]
 
-      def self.from(response, key)
-        new(response.status, reference_data: response.body[key])
+      def self.from(response)
+        data = response.body['country_list'].map do |c|
+          {
+            'country_name' => c['country_name'],
+            'country_code_iso3' => c['country_code_iso3']
+          }
+        end
+        new(response.status, countries: data)
+      end
+    end
+
+    class StatesResponse < Response
```

Comment:
I'd be +1 for moving these into their own files.  I like being able to grep for files by class name, when trying to debug or trace things, in 6 months from now (or by another dev) type thing.  

That said, I understand that these are all related, and small, and see why you put all three classes in one file.  Your call!

Reply:
I agree. Moving them to their own files is probably better practice. I'll take care of that shortly.